docs/modules/plugins/pages/plugin-mobile-app.adoc

```diff
@@ -425,6 +425,27 @@ Then number of elements found by `xpath(html)` is equal to `1`
 ----
 
 
+=== Switch to Web view by name
+
+Switches context to a web view with name that matches the rule. See http://appium.io/docs/en/writing-running-appium/web/hybrid/[Automating hybrid apps] for more information.
```

Comment:
pleaes reuse this 'See http://appium.io/docs/en/writing-running-appium/web/hybrid/[Automating hybrid apps] for more information.'